website/source/guides/stateful-workloads/host-volumes.md

```diff
@@ -0,0 +1,338 @@
+### Step 4: Deploy the MySQL Database
+
+Register the job file you created in the previous step with the following
+command:
+
+```
+$ nomad run mysql.nomad 
+==> Monitoring evaluation "aa478d82"
+    Evaluation triggered by job "mysql-server"
+    Allocation "6c3b3703" created: node "be8aad4e", group "mysql-server"
+    Evaluation status changed: "pending" -> "complete"
+==> Evaluation "aa478d82" finished with status "complete"
+```
+
+Check the status of the allocation and ensure the task is running:
+
+```
+$ nomad status mysql-server
+ID            = mysql-server
+...
+Summary
+Task Group    Queued  Starting  Running  Failed  Complete  Lost
+mysql-server  0       0         1        0       0         0
+```
+
+### Step 5: Connect to MySQL 
+
+Using the mysql client (installed in [Prerequisite 1]), connect to the database
+and access the information:
+
+```
+mysql -h mysql-server.service.consul -u web -p -D itemcollection
+```
+The password for this demo database is `password`.
+
+~> **Please Note:** This guide is for demo purposes and does not follow best
+practices for securing database passwords. See [Keeping Passwords
+Secure][password-security] for more information.
+
+Consul is installed alongside Nomad in this cluster so we were able to
+connect using the `mysql-server` service name we registered with our task in
+our job file.
+
+### Step 6: Add Data to MySQL
+
+Once you are connected to the database, verify the table `items` exists:
+
+```
+mysql> show tables;
++--------------------------+
+| Tables_in_itemcollection |
++--------------------------+
+| items                    |
++--------------------------+
+1 row in set (0.00 sec)
+```
+
+Display the contents of this table with the following command:
+
+```
+mysql> select * from items;
++----+----------+
+| id | name     |
++----+----------+
+|  1 | bike     |
+|  2 | baseball |
+|  3 | chair    |
++----+----------+
+3 rows in set (0.00 sec)
+```
+
+Now add some data to this table (after we terminate our database in Nomad and
+bring it back up, this data should still be intact):
+
+```
+mysql> INSERT INTO items (name) VALUES ('glove');
+```
+
+Run the `INSERT INTO` command as many times as you like with different values.
+
+```
+mysql> INSERT INTO items (name) VALUES ('hat');
+mysql> INSERT INTO items (name) VALUES ('keyboard');
+```
+Once you you are done, type `exit` and return back to the Nomad client command
+line:
+
+```
+mysql> exit
+Bye
+```
+
+### Step 7: Stop and Purge the Database Job
+
+Run the following command to stop and purge the MySQL job from the cluster:
+
+```
+$ nomad stop -purge mysql-server
+==> Monitoring evaluation "6b784149"
+    Evaluation triggered by job "mysql-server"
+    Evaluation status changed: "pending" -> "complete"
+==> Evaluation "6b784149" finished with status "complete"
+```
+
+Verify no jobs are running in the cluster:
+
+```
+$ nomad status
+No running jobs
+```
+You can optionally stop the nomad service on whichever node you are on and move
```

Comment:
this wouldn't necessarily work with host volumes unless they had external data replication

Reply:
I've resolved the other issues, but I'm not clear on this particular issue. Can this line simply be removed for accuracy or is there another issue I'm missing?

Reply:
I missed this line in the sweep. 🤦‍♂ Deleted it; added new copy to explain how HA might be supported by a mount over the directory in question.